src/Expensify.tsx

```diff
@@ -304,30 +303,4 @@ function Expensify({
 
 Expensify.displayName = 'Expensify';
 
-export default withOnyx<ExpensifyProps, ExpensifyOnyxProps>({
-    isCheckingPublicRoom: {
-        key: ONYXKEYS.IS_CHECKING_PUBLIC_ROOM,
-        initWithStoredValues: false,
-    },
-    updateAvailable: {
-        key: ONYXKEYS.UPDATE_AVAILABLE,
-        initWithStoredValues: false,
-    },
-    updateRequired: {
-        key: ONYXKEYS.UPDATE_REQUIRED,
-        initWithStoredValues: false,
-    },
```

Comment:
@danieldoglas I think there's a very likely possibility this change has been leading to issues with the `updateRequired` logic. The intended behavior is that this value will only be `true` when it is explicitly set to true AFTER the component has mounted. If the value gets stuck on `true` then the user will get stuck and be unable to upgrade since they'd be on the correct version already, but have out of date Onyx values.